sourcegen-generator-java/src/main/java/io/micronaut/sourcegen/JavaPoetSourceGenerator.java

```diff
@@ -593,9 +651,9 @@ private CodeBlock renderExpression(@Nullable ObjectDef objectDef, MethodDef meth
             if (memberElement instanceof MethodElement methodElement) {
                 return renderExpression(objectDef, methodDef, getPropertyValue.instance().invoke(methodElement));
             }
-            if (memberElement instanceof FieldElement fieldElement) {
+            // if (memberElement instanceof FieldElement fieldElement) {
                 // TODO: support field
-            }
+            // }
```

Comment:
This was not implemented?

Reply:
It came from master, so I did not want to touch it.